### PR TITLE
Remove beta article "Grant Access to Data in Audit Indexes'

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request removes this beta article:
https://help.sumologic.com/docs/manage/security/audit-indexes/audit-index-access/

Since the new "RBAC for Indexes" feature will replace this old beta feature, we'll publish the PR when the "RBAC for Indexes" article goes live (with PR #3393):
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes/

Issue: Asana ticket "[RBAC for Indexes - Remove beta article "Grant Access to Data in Audit Indexes](https://app.asana.com/0/1183741177809740/1205712234148895/f)""

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205712234148895

### CODE REVIEW

**Review by kimsauce — CHANGES_REQUESTED** [review]
question about redirecting an existing doc

File: cid-redirects.json [hunk 1]
```diff
@@ -3830,7 +3830,8 @@
   "/docs/manage/security/audit-index": "/docs/manage/security/audit-indexes/audit-index",
   "/docs/manage/security/audit-event-index": "/docs/manage/security/audit-indexes/audit-event-index",
   "/docs/manage/security/search-audit-index": "/docs/manage/security/audit-indexes/search-audit-index",
-  "/docs/manage/security/audit-index-access": "/docs/manage/security/audit-indexes/audit-index-access",
+  "/docs/manage/security/audit-index-access": "/docs/manage/users-roles/roles/rbac-for-indexes",
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply cid-redirects.json:3833]: @kimsauce, that isn't a parent page. It's the old location of the beta article before a reworking of the Audit Indexes section.

[the remark this answers is not in the document]
Comment (jpipkin1) [reply cid-redirects.json:3833]: That last one is the most recent location of the beta to be removed. We can do a quick Zoom or Slack huddle if you need me to explain it

**Review by kimsauce — APPROVED** [review]

File: cid-redirects.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply cid-redirects.json:3833]: thank you for explaining this! gotcha.

**Comment by jpipkin1** [comment]
Closing this PR because it's replaced by this commit on PR #3393: "[Remove 'Grant Access to Data in Audit Indexes' article](https://github.com/SumoLogic/sumologic-documentation/pull/3393/commits/c38e2f0c82dea705ba4c800dff73a4153d25e71a)"

**Pull request closed without merging**